decidim-generators/lib/decidim/generators/app_templates/secrets.yml.erb

```diff
@@ -21,27 +52,37 @@ elections_default: &elections_default
   number_of_trustees: 3
 
 default: &default
+  decidim:
+    <<: *decidim_default
   omniauth:
     facebook:
-      # It must be a boolean. Remember ENV variables doesn't support booleans.
-      enabled: false
+      # ENV variables doesn't support booleans.
+      enabled: <%%= ENV["OMNIAUTH_FACEBOOK_APP_ID"].present? ? "true" : "false" %>
```

Comment:
I think this would not work:
```ruby
irb(main):006:0> true == "true"
=> false

```
We would need to also change the check in https://github.com/decidim/decidim/blob/012b5b1342824f5d5566d9aad0fb55705e8be8f8/decidim-core/app/models/decidim/omniauth_provider.rb#L11 - taking into account that this could be the true boolean or a string with value "true"

Reply:
It should work, in fact is being tested in https://github.com/decidim/decidim/blob/abf2c3e911fcef4293354fbf6c4d8bb62ba5d967/decidim-generators/lib/decidim/generators/test/generator_examples.rb#L88
Keep in mind that the "true"/"false" string is being rendered in a YAML file (which converts them into boolean when parsed).

Reply:
Oh, if this is rendered then it shouldn't be an issue, great!